src/components/backend-ai-session-list.ts

```diff
@@ -1434,7 +1451,7 @@ export default class BackendAiSessionList extends BackendAIPage {
             <mwc-icon-button class="fg red controls-running"
                                icon="power_settings_new" @click="${(e) => this._openTerminateSessionDialog(e)}"></mwc-icon-button>
           ` : html``}
-          ${(this._isRunning && !this._isPreparing(rowData.item.status)) || this._APIMajorVersion > 4 ? html`
+          ${(this._isRunning && !this._isPreparing(rowData.item.status)) && this._APIMajorVersion > 4 ? html`
```

Comment:
It is intended to display the log button always for running and preparing, but not pending, sessions when the api version is greater than 4. Is there a reason to change this?

Reply:
@adrysn
Until now, if API version is upper than 4, then log icon has been enabled whether the session is `PENDING` status or not. But the session in `PENDING` status, which is corresponding to the container about to be created, will bring an error when it calls log from the manager server. I think the log icon should be `disabled` or at least we should provide more detailed error msg, such as "there is no corresponding container".

Reply:
<img width="856" alt="Screen Shot 2022-01-10 at 9 10 17 PM" src="https://user-images.githubusercontent.com/46954439/148764003-56557315-6a70-4a28-8543-11edcea5efba.png">